core/arch/arm/arm.mk

```diff
@@ -140,6 +140,15 @@ ifeq ($(CFG_ARM32_core),y)
 # CFG_DT_ADDR:       if defined, forces Device Tree data physical address.
 endif
 
+# CFG_OUTER_CACHE_LINE_SHIFT is used to define platform specific outer cache
```

Comment:
I'd prefer `CFG_MAX_CACHE_LINE_SHIFT` since this is what we care about here.

Reply:
OK

Reply:
Updated to PR.